assets/javascript/game.js

```diff
@@ -0,0 +1,99 @@
+    var targetScore;
+    var buttonNums = []; //variable to store array of random values for buttons
+    var playerScore; //We could set this to 0 and reset at quit or end instead, but using at start is more consistent
+    var wins = 0;
+    var losses = 0;
+
+    
+    function randButtons() {
+        var buttonValues = [];
+        for (i=0; i<4; i++) {
+            buttonValues[i] = Math.floor((Math.random() * 11) + 1);
+        }
+        return buttonValues;
+    }
+
+    function initialize() { //Since there are many different points at which the game will be reset, having a function for that is useful
+        gamestart = false; // Set our state back to before start
+        targetScore = undefined; //Initializing from both ends for extra certainty of values
+        buttonNums = [];
+        $("#target").text("Target Score: ");
+        $("#player").text("Your Score: ");
+        $('#crystalSkullCap').text('');
+        $('#humanSkullCap').text('');
+        $('#lostSoulCap').text('');
+        $('#skullAndCrossbonesCap').text('');
+    }
+
+    //Function to check if player has won or lost
+    function judgement(score) { //Use playerScore indirectly as an input to avoid accidental changes
+        if (score === targetScore) { //The winning condition
+            alert("You have won…the Skull 'O Rama!");
+            wins++;
+            $("#wins").text("Wins: " + wins);
+            initialize();
+        }
+        else if (score > targetScore) { //The losing condition
+            alert("You have failed me! (but not for the last time)");
+            losses++;
+            $("#losses").text("Losses: " + losses);
+        }
+    }
+
+    //Functions for image clicks
+    $("#crystalSkull").click(function() {
+        if(gamestart) {
+            playerScore += buttonNums[0];
+            $("#player").text("Your Score: " + playerScore);
+            $("#crystalSkullCap").text(buttonNums[0]);
+            judgement(playerScore);
+        }
+    })
+
+    $("#humanSkull").click(function() {
+        if(gamestart) {
+            playerScore += buttonNums[1];
+            $("#player").text("Your Score: " + playerScore);
+            $("#humanSkullCap").text(buttonNums[1]);
+            judgement(playerScore);
+        }
+    })
+
+    $("#lostSoul").click(function() {
+        if(gamestart) {
+            playerScore += buttonNums[2];
+            $("#player").text("Your Score: " + playerScore);
+            $("#lostSoulCap").text(buttonNums[2]);
+            judgement(playerScore);
+        }
+    })
+
+    $("#skullAndCrossbones").click(function() {
+        if(gamestart) {
+            playerScore += buttonNums[3];
+            $("#player").text("Your Score: " + playerScore);
+            $("#skullAndCrossbonesCap").text(buttonNums[3]);
+            judgement(playerScore);
+        }
+    })
+
+    //Start and Quit buttons
+    $("#start").click(function() {
```

Comment:
I recommend your start button/box to display starting score instead of appending a random number into the div to let the user see their actual score instead of making them scroll down more after pressing the "start" button.